x-pack/plugins/apm/public/components/shared/aggregated_transactions_callout/index.tsx

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import React from 'react';
+import { EuiCallOut } from '@elastic/eui';
+import { i18n } from '@kbn/i18n';
+
+export function AggregatedTransactionsCallout() {
+  return (
+    <EuiCallOut
+      title={i18n.translate(
+        'xpack.apm.useAggregatedTransactions.callout.title',
+        {
+          defaultMessage:
+            'Transactions events are used to display data in this page',
```

Comment:
I think we should only show the callout if the UI is _falling back to transactions_. 
So if the user changes `xpack.apm.searchAggregatedTransactions = never` we're currently showing the callout always. I think we should only show it when `searchAggregatedTransactions` is `auto` or `always`, and we happen to use transactions.